apps/examples/ble_tester/ble_tester_main.c

```diff
@@ -1001,6 +1001,15 @@ int ble_tester_main(int argc, char *argv[])
 			} else {
 				RMC_LOG(RMC_SERVER_TAG, "[INDI] Send Indi OK\n");
 			}
+		} else if (strncmp(argv[1], "test8", 5) == 0) {
```

Comment:
i think, It seems safe to check until the last null.
```suggestion
		} else if (strncmp(argv[1], "test8", 6) == 0) {
```